docs(upload): Improve KB example (#1852)
telerik/blazor-docs: knowledge-base/upload-files-in-grid.md

1. [Implement the Upload controller methods]({%slug upload-overview%}#implement-controller-methods), which receive and delete the uploaded files. File deletion is optional.
1. Handle the [`OnSuccess` event of the Upload]({%slug upload-events%}#onsuccess) to confirm successful uploads or file deletions, and update the Grid data item, which is the `<EditorTemplate>` `context`.

> 1. The name of the saved file on the server can depend on the Razor UI or on the controller. If it depends on the UI, send it to the controller via the `OnUpload` event arguments (`args.RequestData`). If the file name depends on the controller, receive it in the `OnSuccess` event arguments via `args.Request.ResponseText`.

Reviewer: ```suggestion
1. Depending on your implementation, send or receive the name of the saved file:
   * If the name of the saved file depends on the Razor UI, send it to the controller via the `OnUpload` event arguments (`args.RequestData`).
   * If the If the file name depends on the controller, receive it in the `OnSuccess` event arguments via `args.Request.ResponseText`.
```

> 1. Handle the [Grid `OnUpdate`, `OnCreate` and `OnDelete` events]({%slug components/grid/editing/overview%}#events) to commit changes to the Grid data source. Optionally, delete the respective saved files in `OnDelete`. The example below uses `OnAdd` to provide the `Id` of the new Grid data item, which also affects the uploaded file's name.

Reviewer: The first step on [line 42](https://github.com/telerik/blazor-docs/pull/1852/files#diff-0a63522aef753947e6b734cfcea5e1a9491ade7911697a1e1c44a97e6ab1344cR42) isn't a step that is a part of the proposed solution. Furthermore, if the client takes it to heart, it will render the rest of the steps irrelevant. Consider making this step a note and moving it after the steps in the solution.

Author: Actually, step 1 is a prerequisite for the remaining steps :) That's why I replaced "consider" with "use"
